### DISCUSSION TITLE
这两天一直不能用啊   什么问题

### DISCUSSION THREAD

**Discussion by dwei1002** [community/community#49713] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

这两天一直是打不开软件，要么提示系统维护，要么提示账户异常，能不能用啊  不能用要么退钱 要么升级处理

**Comment by queenofcorgis** [comment]
Hi @dwei1002 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
